index.html

```diff
@@ -0,0 +1,264 @@
+        </header>
+        <div class="features__main container">
+            <div class="feature__item feature__item-left cell">
+                <div class="feature__icon-container"><img
+                        class="feature__icon"
+                        src="./images/responsive.svg"
+                        alt=""
+                    ></div>
+                <h3 class="feature__title">fully responsive</h3>
+                <p class="feature_text">Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
+            </div>
+            <div class="feature__item feature__item-middle cell">
+                <div class="feature__icon-container"><img
+                        class="feature__icon"
+                        src="./images/customizable.svg"
+                        alt=""
+                    ></div>
+                <h3 class="feature__title">customizable</h3>
+                <p class="feature_text">Change the colors, pictures or any of the sections to suit your needs.</p>
+            </div>
+            <div class="feature__item feature__item-right cell">
+                <div class="feature__icon-container"><img
+                        class="feature__icon"
+                        src="./images/design.svg"
+                        alt=""
+                    ></div>
+                <h3 class="feature__title">slick and beautiful design</h3>
+                <p class="feature_text">Trendy and fresh design, fits any website.</p>
+            </div>
+        </div>
+    </section>
+    <section class="logos row">
+        <div class="logos__container container ">
+            <img
+                src="./images/logos.png"
+                alt=""
+                class="logos__item "
+            >
+        </div>
+    </section>
+    <main class="main row">
+        <div class="main__container">
+            <div class="photo__container cell"><img
+                    src="./images/imac.png"
+                    alt=""
+                ></div>
+            <article class="main__text cell">
+                <h2 class="main__heading">simple and beautiful</h2>
+                <p class="main__paragraph">Use Nova theme for your next web project. It is completely customizable so
+                    you
+                    can change any of the
+                    sections to fit your needs. Nova Theme is Free for any kind of use, personal and commercial. Have
+                    fun
+                    and good luck!</p>
+            </article>
+        </div>
+    </main>
+    <section class="pricing row">
+        <header class="pricing__header container">
+            <h2 class="pricing__heading-top">choose your pricing plan</h2>
+            <p class="pricing__heading-paragraph">Pick any of our super affordable pricing plans
+            </p>
+        </header>
+        <section class="pricing__items container">
+            <div class="pricing__item pricing__item-sides cell">
```

Comment:
Tutaj nawet zamiast `div` można dać `article`, bo każdy pakiet cenowy to jedna spójna treść.